cmd/k3d/mkcert.go

```diff
@@ -48,7 +48,7 @@ func mkCert(cmd *cobra.Command, args []string) error {
 
 	err = k3d.GenerateSingleTLSSecret(kcfg.Clientset, *config, appNameFlag, appNamespaceFlag)
 	if err != nil {
-		return fmt.Errorf("error generating certificate for %s/%s: %s", appNameFlag, appNamespaceFlag, err)
+		return fmt.Errorf("error generating certificate for %s/%s: %w", appNameFlag, appNamespaceFlag, err)
```

Comment:
To better support errors in the future and checking them, this should've been a wrapped error with `%w` instead of `%s`.